charts/vc-authn-oidc/templates/deployment.yaml

```diff
@@ -39,6 +39,13 @@ spec:
         - name: controller-config
           configMap:
             name: {{ include "global.fullname" . }}-controller-config
+        {{- if .Values.controller.htmlFileOverrides }}
+        - name: controller-templates
+          projected:
+            sources:
+              - configMap:
+                  name: {{ include "global.fullname" . }}-template-overrides
+        {{- end }}
```

Comment:
This would be toggled by `.Values.controller.customHtmlConfigMapName`: if a name is specified, the name of the configmap would be `.Values.controller.customHtmlConfigMapName`